reference/8-ed-fi-certification/2-certification-requirements-for-data-providers/readme.md

```diff
@@ -0,0 +1,30 @@
+# Certification Requirements for Data Providers
```

Comment:
decided to omit all template files from certification requirements as I don't think we'd want these large excel or word docs in the repo

Reply:
Yes that's true... though we can have links back to the files in Confluence, for now.